### PR TITLE
docs: fix view plugin config at quickstart

### PR DESCRIPTION
**Pull request by atian25**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

- docs: fix view plugin config at quickstart, close https://github.com/eggjs/egg/issues/481
- remove unused fixtures
- improve egg cov

### CODE REVIEW

File: docs/source/zh-cn/advanced/view-plugin.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ module.exports = {
 
 框架本身提供了 `ctx.helper` 供开发者使用，但有些情况下，我们希望对 helper 方法进行覆盖，仅在模板渲染时生效。
 
-如 `helper.shtml` 在 controller 中使用时直接输出清洗后的 html，但在 [nunjucks] 模板中需要绕过模板引擎本身的 `escape` 机制，则可以如下：
+如 `helper.shtml` 在 controller 中使用时，已经清洗过 html，在 nunjucks 模板无需再经过模板引擎本身的 `escape` 机制，则可以如下：
```
Comment (popomore) [inline docs/source/zh-cn/advanced/view-plugin.md]: 改了后反而不太通顺

Comment (atian25) [inline docs/source/zh-cn/advanced/view-plugin.md]: 改了，再看看

**Comment by atian25** [comment]
本地跑测试是通过的, travis 日志点击过去看不到, 500 了.

**Comment by atian25** [comment]
ci 过了，@popomore 再看看

**Pull request merged**